ddtrace/settings/_otel_remapper.py

```diff
@@ -132,6 +132,29 @@ def _remap_default(otel_value: str) -> Optional[str]:
 }
 
 
+# https://github.com/open-telemetry/opentelemetry-python/blob/v1.34.1/opentelemetry-sdk/src/opentelemetry/sdk/environment_variables/__init__.py
+SUPPORTED_OTEL_ENV_VARS = {
```

Comment:
Can we add an otel config test to verify that setting these configs doesn't log a warning. If we use a subprocess test it should be really straightforward 

@pytest.mark.subprocess(ddtrace_run=True, env={supported otel envs})
def test_supported_otel_envs_dont_log_warnings(): 
   pass # or whatever assertions you want. Subprocess tests fail is a warning is logged so you don't NEED to add anything here.

Reply:
We already have individual tests in `tests/opentelemetry/test_config.py` to test each of the variables in SUPPORTED_OTEL_ENV_VARS, and in this PR I added one more for the new combination of `OTEL_METRICS_EXPORTER=otlp`, so I think we're set